### PR TITLE
build: upgrade to node 16 and npm 8

### PR DESCRIPTION
**Pull request by ghassanmas**
Things that have be done:
- Creating .nvmrc file with v16
- Changing .github ci.yml to use node version from nvmrc instead
 of hard coding the version
- Run fresh npm i to reflect the changes on pacakge-lock

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @ghassanmas! I've created [OSPR-6289](https://openedx.atlassian.net/browse/OSPR-6289) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by ghassanmas** [comment]
@natabene this and [this](https://github.com/edx/frontend-app-learning/pull/771) are ready from my side. And I am aware both are waiting for edx internal stuff hence [this comment](https://github.com/edx/frontend-app-profile/pull/491#issuecomment-990024936)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@ jobs:
       with:
         fetch-depth: 0
     - name: Setup Nodejs
-      uses: actions/setup-node@v2
+      uses: dcodeIO/setup-node-nvm@v4
       with:
-        node-version: 12
+        node-version: "${{ env.NODE_VER }}"
```
Comment (davidjoy) [inline .github/workflows/ci.yml:14]: Check out this line from the frontend-platform PR: https://github.com/edx/frontend-platform/pull/259/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR19

We need to set NODE_VER from .nvmrc like it does there.

**Review by davidjoy — CHANGES_REQUESTED** [review]
This PR should also update the release.yml.  You can use this PR as a guide for the changes necessary: https://github.com/edx/frontend-platform/pull/259

**Comment by binodpant** [comment]
@ghassanmas I don't think we need to use this approach anymore, please check https://openedx.atlassian.net/wiki/spaces/AC/pages/3318054984/Node+16+Upgrade and if there is a bot PR we can use that as well

**Comment by binodpant** [comment]
> So shall this be closed and be handled by ArbiBOM?also there is other PR in learning with similar situation

that's correct @ghassanmas 

I am going to ping @Jawayria since it looks like the bot did not create a PR for this repo, let's use that approach instead 

You can go ahead and close this PR out

Thanks!

**Comment by edx-webhook** [comment]
@ghassanmas Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-component-header&entry.867055334=https://github.com/openedx/frontend-component-header/pull/161&entry.1484655318=https://github.com/ghassanmas&entry.752974735=2021-12-16+13:17&entry.1917517419=2022-03-24+20:12&entry.2133058324=No) so we can improve your experience in the future.

**Comment by Jawayria** [comment]
> > So shall this be closed and be handled by ArbiBOM?also there is other PR in learning with similar situation
> 
> that's correct @ghassanmas
> 
> I am going to ping @Jawayria since it looks like the bot did not create a PR for this repo, let's use that approach instead
> 
> You can go ahead and close this PR out
> 
> Thanks!

Hi @binodpant @ghassanmas we've already added Node 16 support in frontend-component-header.

**Pull request closed without merging**